snuba/migrations/snuba_migrations/querylog/0003_add_profile_fields.py

```diff
@@ -0,0 +1,112 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    Array,
+    Column,
+    LowCardinality,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.MultiStepMigration):
+    """
+    Adds fields for query profile.
+    """
+
+    blocking = True
+
+    def __forward_migrations(self, table_name: str) -> Sequence[operations.Operation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.QUERYLOG,
+                table_name=table_name,
+                column=Column(
+                    "clickhouse_queries.all_columns",
+                    Array(Array(LowCardinality(String()))),
```

Comment:
Can you explain why you made these an array of arrays? Just seems like an odd choice.

Reply:
I added an explanation in the comment on querylog.py.
`Array(Array(LowCardinality(String)))` here means:
- the clickhouse_queries array. There are multiple clickhosue queries per snuba query
- the array of columns in the specific clickhouse query in the array above. 

To my knowledge this is the only syntax to add columns to a nested data structure since the nested construct is only syntactic sugar over arrays of arrays.